lib/ansible/modules/crypto/get_certificate.py

```diff
@@ -24,11 +24,12 @@
         - The host to get the cert for (IP is fine)
       type: str
       required: true
-    ca_certs:
+    ca_cert:
       description:
-        - A PEM file containing a list of root certificates; if present, the cert will be validated against these root certs.
+        - A PEM file containing one or more root certificates; if present, the cert will be validated against these root certs.
         - Note that this only validates the certificate is signed by the chain; not that the cert is valid for the host presenting it.
       type: path
+      aliases: [ ca_certs ]
```

Comment:
Since this module just has been added for 2.8, how about removing the alias and just keeping `ca_cert`? @MarkusTeufelberger @john-westcott-iv

Reply:
That's even better as long as there are no objections.

Reply:
I'll create a follow-up PR for this once this is merged.